### PR TITLE
Populate political seeds file.

### PR DESCRIPTION
**Pull request by cody-code-wy**
added political figures

- Donald Trump
- Hillary Clinton
- Bill Clinton
- Bernie Sanders

### CODE REVIEW

**Comment by monicamow** [comment]
Hi William, I would like a more detailed commit message :)
For consideration - please indicate where this file gets required (seeds.rb) and then say that seeds.rb is part of rake task

File: db/political_seeds.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
+topic = Topic.create!(topic: "Political")
+
+# Real trump account
+
+trump = TwitterHandle.create!(
+  topic: topic,
+  twitter_handle: "realdonaldtrump"
+)
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
+topic = Topic.create!(topic: "Political")
+
+# Real trump account
+
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  topic: topic,
+  twitter_handle: "realdonaldtromp",
+  real_twitter_handle_id: trump.id
+)
+
+TwitterHandle.create!(
+  topic: topic,
+  twitter_handle: "donnytrumpy",
+  real_twitter_handle_id: trump.id
+)
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  twitter_handle: "hillaryclinton"
+)
+
+# Fake Hillary
+
+TwitterHandle.create!(
+  topic: topic,
+  twitter_handle: "hilliaryclinton",
+  real_twitter_handle_id: hillary.id
+)
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+# Fake Bill
+
+TwitterHandle.create!(
+  topic: topic,
+  twitter_handle: "notbillclinton",
+  real_twitter_handle_id: bill.id
+)
+
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  twitter_handle: "berniesanders"
+)
+
+# Fake Bernie
+
+TwitterHandle.create!(
+  topic: topic,
+  twitter_handle: "berniethoughts",
+  real_twitter_handle_id: bernie.id
+)
```
Comment (monicamow) [inline db/political_seeds.rb]: separate commit

File: db/political_seeds.rb [hunk 7]
```diff
@@ -0,0 +1,79 @@
+topic = Topic.create!(topic: "Political")
+
+# Real trump account
+
+trump = TwitterHandle.create!(
+  topic: topic,
```
Comment (tjukes) [inline db/political_seeds.rb]: This needs to be `topic_id: topic.id`

Comment (cody-code-wy) [inline db/political_seeds.rb]: @tjukes no this is correct according to AR conventions.

**Pull request merged**